pkg/updater/resultstore/resultstore.go

```diff
@@ -53,6 +53,23 @@ func Updater(resultStoreClient *DownloadClient, gcsClient gcs.Client, groupTimeo
 	}
 }
 
+type singleActionTargetResult struct {
```

Comment:
nit: "TargetResult" is a bit confusing to me since this is a collated result. Maybe just "[single|multi]Result" or "[single|multi]Action"?

Reply:
"single|multiResult" sounds good to me - I will make the changes

Reply:
scratch that - singleActionResult | multiActionResult imo is better